tests/scm/SCMExecutionOutputUTest.cxxtest

```diff
@@ -231,18 +231,18 @@ void SCMExecutionOutputUTest::test_recursive(void)
 	eval->eval(
 		"(define (royal-incr atom cnt)"
 		"   (cog-set-tv! atom"
-		"       (stv (+ (tv-mean (cog-tv atom)) cnt)"
-		"           (tv-conf (cog-tv atom)))))"
+		"       (stv (+ (cog-tv-mean (cog-tv atom)) cnt)"
```

Comment:
are the changes made from here on till the last one, also incorrect?

Reply:
Hmm. Its ... well, its a coin-flip.  The unit test defined a custom version of `tv-mean`, which just happened to do exactly the same thing as the "official" version.  The reason for doing this is because the unit test was old, and it was trying to avoid having to load `utilities.scm`.  What should we do now? There are two choices:

1) continue to define a custom version of `tv-mean` and use that customer version in the body of the test.

2) Remove the definition of `tv-mean` and use `cog-tv-mean` in the body of the text.

Either 1) or 2) is OK, I guess. I have a slight preference for 1) because it tests a more obscure code path, which is probably not tested anywhere else. On the other hand, 2) makes the unit test easier to understand.  So its a coin-flip.

Reply:
BTW, you have run the unit tests, and they pass, right?